static/app/views/explore/contexts/spanTagsContext.tsx

```diff
@@ -12,6 +12,47 @@ import usePrevious from 'sentry/utils/usePrevious';
 import {SpanIndexedField} from 'sentry/views/insights/types';
 import {useSpanFieldCustomTags} from 'sentry/views/performance/utils/useSpanFieldSupportedTags';
 
+export const SentryStringTags = [
```

Comment:
Let's be more clear what this is and follow conventions

```suggestion
export const SENTRY_SPAN_STRING_TAGS = [
```